app/forms/tufts/has_transcript_form.rb

```diff
@@ -0,0 +1,12 @@
+module Tufts
+  class HasTranscriptForm < Hyrax::Forms::WorkForm
```

Comment:
Could this be written as a module instead of a class?  It's possible that's a terrible idea...